internal/hammer/loadtest/workers.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/transparency-dev/merkle/proof"
 	"github.com/transparency-dev/merkle/rfc6962"
 	"github.com/transparency-dev/static-ct/internal/client"
-	"github.com/transparency-dev/static-ct/internal/types"
+	rfc69621 "github.com/transparency-dev/static-ct/internal/types/rfc6962"
```

Comment:
```suggestion
	"github.com/transparency-dev/static-ct/internal/types/rfc6962"
```

Reply:
Good catch. I can't do this because of a different import, but I sorted this out.